macOS/LocalPackages/DataBrokerProtectionShared/Sources/DataBrokerProtectionShared/Operations/DataBrokerJob.swift

```diff
@@ -53,19 +53,6 @@ public protocol DataBrokerJob: CCFCommunicationDelegate {
     func executeCurrentAction() async
 }
 
-public extension DataBrokerJob {
-    func run(inputValue: InputValue,
-             webViewHandler: WebViewHandler?,
-             actionsHandler: ActionsHandler?,
-             shouldRunNextStep: @escaping () -> Bool) async throws -> ReturnValue {
-
-        try await run(inputValue: inputValue,
-                      webViewHandler: webViewHandler,
-                      actionsHandler: actionsHandler,
-                      showWebView: false)
-    }
-}
```

Comment:
I wasn't able to find anywhere that this extension was used either, and everything seems to compile, so I've taken it out as well - let me know if we'd prefer to keep it in.

Reply:
There's generally quite a lot of these helper extensions that IMO mostly just complicate the public interface so we should strive to remove